### ISSUE TITLE
Incorrect mapping of some integer types

### ISSUE THREAD

**Issue by hsolbrig** [fhircat/org.hl7.fhir.core#2]
The positiveInt fhir type need to map to xsd:positiveInteger.  See: https://github.com/fhircat/jsonld_context_files/blob/master/contextFiles/positiveint.context.jsonld#L18

We also need to check any other integer flavors

**Comment by cendle** [comment]
I have debugged the code for generating the positiveint.context.jasonld file.

I am grabbing the ElementDefinition.getBase().getPath() value  (in this case it is integer).  I pass this value into the RDFTypeMap to get XSD type.  It returns http://www.w3.org/2001/XMLSchema#integer as I would expect.

I am not sure how I can change this to return positiveInteger...

I did verify that positiveInteger is in the RDFTypeMap.
